lib/Manager.php

```diff
@@ -263,4 +329,55 @@ public function deleteMessagesForSessionIds($sessionIds) {
 			->orWhere($query->expr()->in('sender', $query->createNamedParameter($sessionIds, IQueryBuilder::PARAM_STR_ARRAY)));
 		$query->execute();
 	}
+
+	/**
+	 * @return string
+	 */
+	protected function getNewToken() {
+		$chars = str_replace(['l', '0', '1'], '', ISecureRandom::CHAR_LOWER . ISecureRandom::CHAR_DIGITS);
```

Comment:
Can we exclude 0, 1 and lowercase L? (As you already noted from needing to say »lowercase L«, that’s exactly where the confusion stems from ;)